OD_handler/OD_handler.py

```diff
@@ -0,0 +1,464 @@
+# -*- coding: utf-8 -*-
+"""Main module."""
+
+### Libraries ###
+import pandas as pd
+from datetime import datetime
+import croissance
+from croissance import process_curve
+from croissance.estimation.outliers import remove_outliers
+import re
+import os
+import matplotlib.pyplot as plt 
+import matplotlib
+import numpy as np
+from scipy.optimize import curve_fit
+from croissance.estimation.util import with_overhangs
+from croissance.estimation import regression
+from pandas import Series
+import subprocess
+import sys
+from scipy import interpolate
+import math
+import argparse
+
+__version__ = "0.0.1"
+
+
+
+# ------ MAIN RUNNER ------
+
+
+def argument_parser(argv_list=None):
+	'''Asesses the input arguments and outputs flags to run the different functions according to the user needs.
+	Keyword arguments:
+    argv_list -- List of arguments provided by the user when running the program.
+    Returned variables:
+    flags
+	'''
+
+	#Initialize the argument parser
+	parser = argparse.ArgumentParser()
+
+	#Adding
+	parser.add_argument("-e", "--estimations", help="Get only the estimations for every sample", action = "store_true")
```

Comment:
I do not remember if these were in there before, but I can say from experience that they greatly improve the user command line experience.